b'airflow-7877bf75cc-nlkd8\n*** Reading local file: /opt/airflow/logs/db-to-db-valdation-engine/db_to_db_validation_automation/2022-05-27T11:29:27.979748+00:00/1.log\n[2022-05-27 11:29:30,163] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: db-to-db-valdation-engine.db_to_db_validation_automation 2022-05-27T11:29:27.979748+00:00 [queued]>\n[2022-05-27 11:29:30,171] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: db-to-db-valdation-engine.db_to_db_validation_automation 2022-05-27T11:29:27.979748+00:00 [queued]>\n[2022-05-27 11:29:30,171] {taskinstance.py:1095} INFO - \n--------------------------------------------------------------------------------\n[2022-05-27 11:29:30,171] {taskinstance.py:1096} INFO - Starting attempt 1 of 1\n[2022-05-27 11:29:30,171] {taskinstance.py:1097} INFO - \n--------------------------------------------------------------------------------\n[2022-05-27 11:29:30,180] {taskinstance.py:1115} INFO - Executing <Task(KubernetesPodOperator): db_to_db_validation_automation> on 2022-05-27T11:29:27.979748+00:00\n[2022-05-27 11:29:30,186] {standard_task_runner.py:52} INFO - Started process 2347263 to run task\n[2022-05-27 11:29:30,193] {standard_task_runner.py:76} INFO - Running: [\'airflow\', \'tasks\', \'run\', \'db-to-db-valdation-engine\', \'db_to_db_validation_automation\', \'2022-05-27T11:29:27.979748+00:00\', \'--job-id\', \'105\', \'--pool\', \'default_pool\', \'--raw\', \'--subdir\', \'DAGS_FOLDER/db_to_db.py\', \'--cfg-path\', \'/tmp/tmphy9jc2ce\', \'--error-file\', \'/tmp/tmph4e5nnw5\']\n[2022-05-27 11:29:30,195] {standard_task_runner.py:77} INFO - Job 105: Subtask db_to_db_validation_automation\n[2022-05-27 11:29:30,235] {logging_mixin.py:109} INFO - Running <TaskInstance: db-to-db-valdation-engine.db_to_db_validation_automation 2022-05-27T11:29:27.979748+00:00 [running]> on host airflow-7877bf75cc-nlkd8\n[2022-05-27 11:29:30,304] {taskinstance.py:1254} INFO - Exporting the following env vars:\nAIRFLOW_CTX_DAG_OWNER=airflow\nAIRFLOW_CTX_DAG_ID=db-to-db-valdation-engine\nAIRFLOW_CTX_TASK_ID=db_to_db_validation_automation\nAIRFLOW_CTX_EXECUTION_DATE=2022-05-27T11:29:27.979748+00:00\nAIRFLOW_CTX_DAG_RUN_ID=db-to-db-valdation-engine2022-05-2716:59:27.079209\n[2022-05-27 11:29:30,350] {kubernetes_pod.py:366} INFO - creating pod with labels {\'dag_id\': \'db-to-db-valdation-engine\', \'task_id\': \'db_to_db_validation_automation\', \'execution_date\': \'2022-05-27T112927.9797480000-ed9f0cbd9\', \'try_number\': \'1\'} and launcher <airflow.providers.cncf.kubernetes.utils.pod_launcher.PodLauncher object at 0x7f24b7a4ddd8>\n[2022-05-27 11:29:30,548] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:30,549] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:31,558] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:31,558] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:32,571] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:32,572] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:33,714] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:33,714] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:34,724] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:34,725] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:35,734] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:35,735] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:36,744] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:36,744] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:37,754] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:37,754] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:38,765] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:38,765] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:39,776] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:39,776] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:40,789] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:40,789] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:41,800] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:41,800] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:42,810] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:42,811] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:43,821] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:43,821] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:44,830] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:44,830] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:45,840] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:45,840] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:46,851] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:46,851] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:47,860] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:47,860] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:48,868] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:48,869] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:49,877] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:49,878] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:50,887] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:50,888] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:51,901] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:51,901] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:52,909] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:52,910] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:53,921] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:53,921] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:54,932] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Pending\n[2022-05-27 11:29:54,932] {pod_launcher.py:128} WARNING - Pod not yet started: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\n[2022-05-27 11:29:55,942] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Running\n[2022-05-27 11:29:56,158] {pod_launcher.py:149} INFO - /usr/local/lib/python3.8/site-packages/pandas/io/sql.py:761: UserWarning: pandas only support SQLAlchemy connectable(engine/connection) ordatabase string URI or sqlite3 DBAPI2 connectionother DBAPI2 objects are not tested, please consider using SQLAlchemy\n[2022-05-27 11:29:56,158] {pod_launcher.py:149} INFO -   warnings.warn(\n[2022-05-27 11:29:56,158] {pod_launcher.py:149} INFO - db_config_data {\'source_details\': {\'platform\': \'onpremises\', \'name\': \'oracle\', \'connection_details\': {\'password\': \'supportrdonly321\', \'username\': \'SUPPORTRDONLY\', \'port\': \'1521\', \'host\': \'ora-stage-ebs-r1-vip.vmware.com\', \'service_name\': \'VMWSTAGE\', \'drivername\': \'oracle.jdbc.driver.OracleDriver\', \'database\': \'oracle\'}, \'additional_details\': {\'table name\': \'\'}}, \'target_details\': {\'platform\': \'onpremises\', \'name\': \'presto\', \'connection_details\': {\'password\': \'W.8u8bRe^d^!u1Y3^VP\', \'username\': \'svc.presto_ob_dev\', \'port\': \'8443\', \'host\': \'prestosql-stg.tkg.vmware.com\', \'database\': \'edw_tesseract\'}, \'additional_details\': {\'tablename\': {}}}, \'output_details\': {\'platform\': \'onpremises\', \'name\': \'postgresql\', \'connection_details\': {\'password\': \'mysecretpassword\', \'username\': \'postgres\', \'port\': \'5432\', \'host\': \'10.168.3.214\', \'database\': \'output_dataset\'}, \'additional_details\': {\'table name\': \'db_to_db_validation_output_final\'}}, \'metrics_details\': {\'platform\': \'onpremises\', \'name\': \'postgresql\', \'connection_details\': {\'password\': \'mysecretpassword\', \'username\': \'postgres\', \'port\': \'5432\', \'host\': \'10.168.3.214\', \'database\': \'output_dataset\'}, \'additional_details\': {\'table name\': \'metrics_db_to_db_final\'}}}\n[2022-05-27 11:29:56,158] {pod_launcher.py:149} INFO - validation_bucket_data {\'data_needed\': {\'source and target\': [\'source_and_target_table_should_be_of_same_length_and_values\', \'source_and_target_table_should_be_of_same_length_and_values\', \'source_and_target_table_should_be_of_same_length_and_values\', \'source_and_target_should_match\', \'source_and_target_should_match\', \'source_and_target_should_match\', \'each_source_and_target_query_should_fetch_some_data\', \'scd_columns_validation\', \'scd_source_target_validation\'], \'only target\': [\'target_query_should_not_return_any_rows\', \'target_query_should_not_return_any_rows\', \'target_table_query_should_fetch_some_data\', \'target_table_query_should_fetch_some_data\', \'table_query_should_fetch_n_number_of_records\', \'table_query_should_fetch_n_number_of_records\', \'target_query_should_not_return_any_rows\', \'target_query_should_not_return_any_rows\', \'target_query_should_not_return_any_rows\', \'scd_target_validation\', \'scd_target_validation\', \'target_query_should_not_return_any_rows\', \'target_query_should_not_contain_any_duplicate_rows\']}, \'validation_buckets\': {\'source_and_target_table_should_be_of_same_length_and_values\': [\'DataCompleteness_Validation\', \'StraightThrough_Column_Validation\', \'Data_Transformation\'], \'source_and_target_should_match\': [\'Structure_Validation\', \'Count_Validation\', \'Numeric_Column\'], \'each_source_and_target_query_should_fetch_some_data\': [\'Data_Availability_Validation\'], \'scd_columns_validation\': [\'SCD_Validation\'], \'target_query_should_not_return_any_rows\': [\'Duplicate_Value_Check\', \'Default_Value_Validation\', \'Empty_Check\', \'Empty_Header_Check\', \'Null_Check\', \'Primary_Key_Column\'], \'target_table_query_should_fetch_some_data\': [\'Date_Column_Validation\', \'Date_Column\'], \'table_query_should_fetch_n_number_of_records\': [\'Count_Value_Validation\', \'Specific_Row_Count_Check\'], \'scd_source_target_validation\': [\'SCD_Type_1_count_check\'], \'scd_target_validation\': [\'SCD_Type_1_Duplicate_Check\', \'SCD_Type_1_Null_Value_Check\'], \'target_query_should_not_contain_any_duplicate_rows\': [\'Duplicate_Check_F2F\']}}\n[2022-05-27 11:29:56,158] {pod_launcher.py:149} INFO - connection established <cx_Oracle.Connection to SUPPORTRDONLY@(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=ora-stage-ebs-r1-vip.vmware.com)(PORT=1521))(CONNECT_DATA=(SERVICE_NAME=VMWSTAGE)))>\n[2022-05-27 11:29:56,158] {pod_launcher.py:149} INFO - source_connection <processor.db_processor.Oracle object at 0x7f28bc8a6760>\n[2022-05-27 11:29:56,158] {pod_launcher.py:149} INFO - connection established <sqlalchemy.engine.base.Connection object at 0x7f28ab0c7970>\n[2022-05-27 11:29:56,159] {pod_launcher.py:149} INFO - target_connection <processor.db_processor.Presto object at 0x7f28ab11e250>\n[2022-05-27 11:29:56,160] {pod_launcher.py:149} INFO - Traceback (most recent call last):\n[2022-05-27 11:29:56,160] {pod_launcher.py:149} INFO -   File "/usr/local/lib/python3.8/site-packages/pandas/io/sql.py", line 2020, in execute\n[2022-05-27 11:29:56,160] {pod_launcher.py:149} INFO -     cur.execute(*args, **kwargs)\n[2022-05-27 11:29:56,160] {pod_launcher.py:149} INFO - cx_Oracle.DatabaseError: ORA-00942: table or view does not exist\n[2022-05-27 11:29:56,160] {pod_launcher.py:149} INFO - \n[2022-05-27 11:29:56,160] {pod_launcher.py:149} INFO - The above exception was the direct cause of the following exception:\n[2022-05-27 11:29:56,160] {pod_launcher.py:149} INFO - \n[2022-05-27 11:29:56,160] {pod_launcher.py:149} INFO - Traceback (most recent call last):\n[2022-05-27 11:29:56,160] {pod_launcher.py:149} INFO -   File "/code/main.py", line 356, in read_test_script_perform_validation\n[2022-05-27 11:29:56,160] {pod_launcher.py:149} INFO -     source_df = source_connection.run_query(current_test_case[const.source_query])\n[2022-05-27 11:29:56,161] {pod_launcher.py:149} INFO -   File "/code/processor/db_processor.py", line 89, in run_query\n[2022-05-27 11:29:56,161] {pod_launcher.py:149} INFO -     df = pd.read_sql(query,self.__conn)\n[2022-05-27 11:29:56,161] {pod_launcher.py:149} INFO -   File "/usr/local/lib/python3.8/site-packages/pandas/io/sql.py", line 566, in read_sql\n[2022-05-27 11:29:56,161] {pod_launcher.py:149} INFO -     return pandas_sql.read_query(\n[2022-05-27 11:29:56,161] {pod_launcher.py:149} INFO -   File "/usr/local/lib/python3.8/site-packages/pandas/io/sql.py", line 2080, in read_query\n[2022-05-27 11:29:56,161] {pod_launcher.py:149} INFO -     cursor = self.execute(*args)\n[2022-05-27 11:29:56,161] {pod_launcher.py:149} INFO -   File "/usr/local/lib/python3.8/site-packages/pandas/io/sql.py", line 2032, in execute\n[2022-05-27 11:29:56,161] {pod_launcher.py:149} INFO -     raise ex from exc\n[2022-05-27 11:29:56,161] {pod_launcher.py:149} INFO - pandas.io.sql.DatabaseError: Execution failed on sql \'SELECT COUNT(*) as table_count FROM  FND_FLEX_VALUE_RULES_VL\': ORA-00942: table or view does not exist\n[2022-05-27 11:29:56,161] {pod_launcher.py:149} INFO - \n[2022-05-27 11:29:56,161] {pod_launcher.py:149} INFO - During handling of the above exception, another exception occurred:\n[2022-05-27 11:29:56,161] {pod_launcher.py:149} INFO - \n[2022-05-27 11:29:56,161] {pod_launcher.py:149} INFO - Traceback (most recent call last):\n[2022-05-27 11:29:56,162] {pod_launcher.py:149} INFO -   File "/code/main.py", line 631, in <module>\n[2022-05-27 11:29:56,162] {pod_launcher.py:149} INFO -     test_script_data_result,metrics_data = read_test_script_perform_validation(source_connection,\n[2022-05-27 11:29:56,162] {pod_launcher.py:149} INFO -   File "/code/main.py", line 450, in read_test_script_perform_validation\n[2022-05-27 11:29:56,162] {pod_launcher.py:149} INFO -     total_source_size+=0\n[2022-05-27 11:29:56,162] {pod_launcher.py:149} INFO - TypeError: \'int\' object is not iterable\n[2022-05-27 11:29:58,207] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Failed\n[2022-05-27 11:29:58,207] {pod_launcher.py:308} ERROR - Event with job id db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 Failed\n[2022-05-27 11:29:58,217] {pod_launcher.py:198} INFO - Event: db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 had an event of type Failed\n[2022-05-27 11:29:58,217] {pod_launcher.py:308} ERROR - Event with job id db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 Failed\n[2022-05-27 11:29:58,301] {taskinstance.py:1463} ERROR - Task failed with exception\nTraceback (most recent call last):\n  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py", line 369, in execute\n    raise AirflowException(f\'Pod {self.pod.metadata.name} returned a failure: {remote_pod}\')\nairflow.exceptions.AirflowException: Pod db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 returned a failure: {\'api_version\': \'v1\',\n \'kind\': \'Pod\',\n \'metadata\': {\'annotations\': None,\n              \'cluster_name\': None,\n              \'creation_timestamp\': datetime.datetime(2022, 5, 27, 11, 29, 30, tzinfo=tzlocal()),\n              \'deletion_grace_period_seconds\': None,\n              \'deletion_timestamp\': None,\n              \'finalizers\': None,\n              \'generate_name\': None,\n              \'generation\': None,\n              \'initializers\': None,\n              \'labels\': {\'airflow_version\': \'2.1.4\',\n                         \'dag_id\': \'db-to-db-valdation-engine\',\n                         \'execution_date\': \'2022-05-27T112927.9797480000-ed9f0cbd9\',\n                         \'kubernetes_pod_operator\': \'True\',\n                         \'task_id\': \'db_to_db_validation_automation\',\n                         \'try_number\': \'1\'},\n              \'managed_fields\': [{\'api_version\': \'v1\',\n                                  \'fields\': None,\n                                  \'manager\': \'OpenAPI-Generator\',\n                                  \'operation\': \'Update\',\n                                  \'time\': datetime.datetime(2022, 5, 27, 11, 29, 30, tzinfo=tzlocal())},\n                                 {\'api_version\': \'v1\',\n                                  \'fields\': None,\n                                  \'manager\': \'kubelet\',\n                                  \'operation\': \'Update\',\n                                  \'time\': datetime.datetime(2022, 5, 27, 11, 29, 56, tzinfo=tzlocal())}],\n              \'name\': \'db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\',\n              \'namespace\': \'ara-adapt-iceberg\',\n              \'owner_references\': None,\n              \'resource_version\': \'1423729336\',\n              \'self_link\': \'/api/v1/namespaces/ara-adapt-iceberg/pods/db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\',\n              \'uid\': \'75ff3d87-f763-4e21-8d96-47a15d41fff6\'},\n \'spec\': {\'active_deadline_seconds\': None,\n          \'affinity\': {\'node_affinity\': None,\n                       \'pod_affinity\': None,\n                       \'pod_anti_affinity\': None},\n          \'automount_service_account_token\': None,\n          \'containers\': [{\'args\': None,\n                          \'command\': None,\n                          \'env\': [{\'name\': \'etl_name\',\n                                   \'value\': \'EBS_FND_FLEX_VALUE_RULES_VL\',\n                                   \'value_from\': None},\n                                  {\'name\': \'bucket_type\',\n                                   \'value\': \'minio\',\n                                   \'value_from\': None},\n                                  {\'name\': \'test_script_file_path\',\n                                   \'value\': \'Unit Testing/Marlin/Automated \'\n                                            \'Test \'\n                                            \'Script/FND_FLEX_VALUE_RULES_VL.csv\',\n                                   \'value_from\': None},\n                                  {\'name\': \'etl_run_id\',\n                                   \'value\': \'EBS_FND_FLEX_VALUE_RULES_VL \'\n                                            \'9853_no_run_2022-05-27 \'\n                                            \'16:59:24.161137\',\n                                   \'value_from\': None},\n                                  {\'name\': \'db_config_json\',\n                                   \'value\': "{\'source_details\': {\'platform\': "\n                                            "\'onpremises\', \'name\': \'oracle\', "\n                                            "\'connection_details\': "\n                                            "{\'password\': \'supportrdonly321\', "\n                                            "\'username\': \'SUPPORTRDONLY\', "\n                                            "\'port\': \'1521\', \'host\': "\n                                            "\'ora-stage-ebs-r1-vip.vmware.com\', "\n                                            "\'service_name\': \'VMWSTAGE\', "\n                                            "\'drivername\': "\n                                            "\'oracle.jdbc.driver.OracleDriver\', "\n                                            "\'database\': \'oracle\'}, "\n                                            "\'additional_details\': {\'table "\n                                            "name\': \'\'}}, \'target_details\': "\n                                            "{\'platform\': \'onpremises\', "\n                                            "\'name\': \'presto\', "\n                                            "\'connection_details\': "\n                                            "{\'password\': "\n                                            "\'W.8u8bRe^d^!u1Y3^VP\', "\n                                            "\'username\': \'svc.presto_ob_dev\', "\n                                            "\'port\': \'8443\', \'host\': "\n                                            "\'prestosql-stg.tkg.vmware.com\', "\n                                            "\'database\': \'edw_tesseract\'}, "\n                                            "\'additional_details\': "\n                                            "{\'tablename\': {}}}, "\n                                            "\'output_details\': {\'platform\': "\n                                            "\'onpremises\', \'name\': "\n                                            "\'postgresql\', "\n                                            "\'connection_details\': "\n                                            "{\'password\': \'mysecretpassword\', "\n                                            "\'username\': \'postgres\', \'port\': "\n                                            "\'5432\', \'host\': \'10.168.3.214\', "\n                                            "\'database\': \'output_dataset\'}, "\n                                            "\'additional_details\': {\'table "\n                                            "name\': "\n                                            "\'db_to_db_validation_output_final\'}}, "\n                                            "\'metrics_details\': {\'platform\': "\n                                            "\'onpremises\', \'name\': "\n                                            "\'postgresql\', "\n                                            "\'connection_details\': "\n                                            "{\'password\': \'mysecretpassword\', "\n                                            "\'username\': \'postgres\', \'port\': "\n                                            "\'5432\', \'host\': \'10.168.3.214\', "\n                                            "\'database\': \'output_dataset\'}, "\n                                            "\'additional_details\': {\'table "\n                                            "name\': "\n                                            "\'metrics_db_to_db_final\'}}}",\n                                   \'value_from\': None},\n                                  {\'name\': \'bucket_config_json\',\n                                   \'value\': "{\'data_needed\': {\'source and "\n                                            "target\': "\n                                            "[\'source_and_target_table_should_be_of_same_length_and_values\', "\n                                            "\'source_and_target_table_should_be_of_same_length_and_values\', "\n                                            "\'source_and_target_table_should_be_of_same_length_and_values\', "\n                                            "\'source_and_target_should_match\', "\n                                            "\'source_and_target_should_match\', "\n                                            "\'source_and_target_should_match\', "\n                                            "\'each_source_and_target_query_should_fetch_some_data\', "\n                                            "\'scd_columns_validation\', "\n                                            "\'scd_source_target_validation\'], "\n                                            "\'only target\': "\n                                            "[\'target_query_should_not_return_any_rows\', "\n                                            "\'target_query_should_not_return_any_rows\', "\n                                            "\'target_table_query_should_fetch_some_data\', "\n                                            "\'target_table_query_should_fetch_some_data\', "\n                                            "\'table_query_should_fetch_n_number_of_records\', "\n                                            "\'table_query_should_fetch_n_number_of_records\', "\n                                            "\'target_query_should_not_return_any_rows\', "\n                                            "\'target_query_should_not_return_any_rows\', "\n                                            "\'target_query_should_not_return_any_rows\', "\n                                            "\'scd_target_validation\', "\n                                            "\'scd_target_validation\', "\n                                            "\'target_query_should_not_return_any_rows\', "\n                                            "\'target_query_should_not_contain_any_duplicate_rows\']}, "\n                                            "\'validation_buckets\': "\n                                            "{\'source_and_target_table_should_be_of_same_length_and_values\': "\n                                            "[\'DataCompleteness_Validation\', "\n                                            "\'StraightThrough_Column_Validation\', "\n                                            "\'Data_Transformation\'], "\n                                            "\'source_and_target_should_match\': "\n                                            "[\'Structure_Validation\', "\n                                            "\'Count_Validation\', "\n                                            "\'Numeric_Column\'], "\n                                            "\'each_source_and_target_query_should_fetch_some_data\': "\n                                            "[\'Data_Availability_Validation\'], "\n                                            "\'scd_columns_validation\': "\n                                            "[\'SCD_Validation\'], "\n                                            "\'target_query_should_not_return_any_rows\': "\n                                            "[\'Duplicate_Value_Check\', "\n                                            "\'Default_Value_Validation\', "\n                                            "\'Empty_Check\', "\n                                            "\'Empty_Header_Check\', "\n                                            "\'Null_Check\', "\n                                            "\'Primary_Key_Column\'], "\n                                            "\'target_table_query_should_fetch_some_data\': "\n                                            "[\'Date_Column_Validation\', "\n                                            "\'Date_Column\'], "\n                                            "\'table_query_should_fetch_n_number_of_records\': "\n                                            "[\'Count_Value_Validation\', "\n                                            "\'Specific_Row_Count_Check\'], "\n                                            "\'scd_source_target_validation\': "\n                                            "[\'SCD_Type_1_count_check\'], "\n                                            "\'scd_target_validation\': "\n                                            "[\'SCD_Type_1_Duplicate_Check\', "\n                                            "\'SCD_Type_1_Null_Value_Check\'], "\n                                            "\'target_query_should_not_contain_any_duplicate_rows\': "\n                                            "[\'Duplicate_Check_F2F\']}}",\n                                   \'value_from\': None},\n                                  {\'name\': \'user\',\n                                   \'value\': \'priyanka\',\n                                   \'value_from\': None},\n                                  {\'name\': \'conn_id\',\n                                   \'value\': \'47\',\n                                   \'value_from\': None},\n                                  {\'name\': \'pipeline_id\',\n                                   \'value\': \'9853\',\n                                   \'value_from\': None}],\n                          \'env_from\': None,\n                          \'image\': \'harbor-repo.vmware.com/tesseract_iceberg/adapt:db2dblatest\',\n                          \'image_pull_policy\': \'Always\',\n                          \'lifecycle\': None,\n                          \'liveness_probe\': None,\n                          \'name\': \'base\',\n                          \'ports\': None,\n                          \'readiness_probe\': None,\n                          \'resources\': {\'limits\': {\'cpu\': \'2\', \'memory\': \'5Gi\'},\n                                        \'requests\': {\'cpu\': \'2\',\n                                                     \'memory\': \'5Gi\'}},\n                          \'security_context\': None,\n                          \'stdin\': None,\n                          \'stdin_once\': None,\n                          \'termination_message_path\': \'/dev/termination-log\',\n                          \'termination_message_policy\': \'File\',\n                          \'tty\': None,\n                          \'volume_devices\': None,\n                          \'volume_mounts\': [{\'mount_path\': \'/var/run/secrets/kubernetes.io/serviceaccount\',\n                                             \'mount_propagation\': None,\n                                             \'name\': \'default-token-76pt8\',\n                                             \'read_only\': True,\n                                             \'sub_path\': None,\n                                             \'sub_path_expr\': None}],\n                          \'working_dir\': None}],\n          \'dns_config\': None,\n          \'dns_policy\': \'ClusterFirst\',\n          \'enable_service_links\': True,\n          \'host_aliases\': None,\n          \'host_ipc\': None,\n          \'host_network\': None,\n          \'host_pid\': None,\n          \'hostname\': None,\n          \'image_pull_secrets\': [{\'name\': \'adapt-secret\'}],\n          \'init_containers\': None,\n          \'node_name\': \'decc-ara-minion033\',\n          \'node_selector\': None,\n          \'preemption_policy\': \'PreemptLowerPriority\',\n          \'priority\': 0,\n          \'priority_class_name\': None,\n          \'readiness_gates\': None,\n          \'restart_policy\': \'Never\',\n          \'runtime_class_name\': None,\n          \'scheduler_name\': \'default-scheduler\',\n          \'security_context\': {\'fs_group\': None,\n                               \'run_as_group\': None,\n                               \'run_as_non_root\': None,\n                               \'run_as_user\': None,\n                               \'se_linux_options\': None,\n                               \'supplemental_groups\': None,\n                               \'sysctls\': None,\n                               \'windows_options\': None},\n          \'service_account\': \'default\',\n          \'service_account_name\': \'default\',\n          \'share_process_namespace\': None,\n          \'subdomain\': None,\n          \'termination_grace_period_seconds\': 30,\n          \'tolerations\': [{\'effect\': \'NoExecute\',\n                           \'key\': \'node.kubernetes.io/not-ready\',\n                           \'operator\': \'Exists\',\n                           \'toleration_seconds\': 300,\n                           \'value\': None},\n                          {\'effect\': \'NoExecute\',\n                           \'key\': \'node.kubernetes.io/unreachable\',\n                           \'operator\': \'Exists\',\n                           \'toleration_seconds\': 300,\n                           \'value\': None}],\n          \'volumes\': [{\'aws_elastic_block_store\': None,\n                       \'azure_disk\': None,\n                       \'azure_file\': None,\n                       \'cephfs\': None,\n                       \'cinder\': None,\n                       \'config_map\': None,\n                       \'csi\': None,\n                       \'downward_api\': None,\n                       \'empty_dir\': None,\n                       \'fc\': None,\n                       \'flex_volume\': None,\n                       \'flocker\': None,\n                       \'gce_persistent_disk\': None,\n                       \'git_repo\': None,\n                       \'glusterfs\': None,\n                       \'host_path\': None,\n                       \'iscsi\': None,\n                       \'name\': \'default-token-76pt8\',\n                       \'nfs\': None,\n                       \'persistent_volume_claim\': None,\n                       \'photon_persistent_disk\': None,\n                       \'portworx_volume\': None,\n                       \'projected\': None,\n                       \'quobyte\': None,\n                       \'rbd\': None,\n                       \'scale_io\': None,\n                       \'secret\': {\'default_mode\': 420,\n                                  \'items\': None,\n                                  \'optional\': None,\n                                  \'secret_name\': \'default-token-76pt8\'},\n                       \'storageos\': None,\n                       \'vsphere_volume\': None}]},\n \'status\': {\'conditions\': [{\'last_probe_time\': None,\n                            \'last_transition_time\': datetime.datetime(2022, 5, 27, 11, 29, 31, tzinfo=tzlocal()),\n                            \'message\': None,\n                            \'reason\': None,\n                            \'status\': \'True\',\n                            \'type\': \'Initialized\'},\n                           {\'last_probe_time\': None,\n                            \'last_transition_time\': datetime.datetime(2022, 5, 27, 11, 29, 56, tzinfo=tzlocal()),\n                            \'message\': \'containers with unready status: [base]\',\n                            \'reason\': \'ContainersNotReady\',\n                            \'status\': \'False\',\n                            \'type\': \'Ready\'},\n                           {\'last_probe_time\': None,\n                            \'last_transition_time\': datetime.datetime(2022, 5, 27, 11, 29, 56, tzinfo=tzlocal()),\n                            \'message\': \'containers with unready status: [base]\',\n                            \'reason\': \'ContainersNotReady\',\n                            \'status\': \'False\',\n                            \'type\': \'ContainersReady\'},\n                           {\'last_probe_time\': None,\n                            \'last_transition_time\': datetime.datetime(2022, 5, 27, 11, 29, 31, tzinfo=tzlocal()),\n                            \'message\': None,\n                            \'reason\': None,\n                            \'status\': \'True\',\n                            \'type\': \'PodScheduled\'}],\n            \'container_statuses\': [{\'container_id\': \'docker://5be95bb4f48b21dac746f488d7ea4e9cd70991785481149fb6bf3563d6cff37c\',\n                                    \'image\': \'harbor-repo.vmware.com/tesseract_iceberg/adapt:db2dblatest\',\n                                    \'image_id\': \'docker-pullable://harbor-repo.vmware.com/tesseract_iceberg/adapt@sha256:d467a80a1942cb13b5e7efbddd4fc30408c439ca5e477ecb25381f4ede6c69d6\',\n                                    \'last_state\': {\'running\': None,\n                                                   \'terminated\': None,\n                                                   \'waiting\': None},\n                                    \'name\': \'base\',\n                                    \'ready\': False,\n                                    \'restart_count\': 0,\n                                    \'state\': {\'running\': None,\n                                              \'terminated\': {\'container_id\': \'docker://5be95bb4f48b21dac746f488d7ea4e9cd70991785481149fb6bf3563d6cff37c\',\n                                                             \'exit_code\': 1,\n                                                             \'finished_at\': datetime.datetime(2022, 5, 27, 11, 29, 56, tzinfo=tzlocal()),\n                                                             \'message\': None,\n                                                             \'reason\': \'Error\',\n                                                             \'signal\': None,\n                                                             \'started_at\': datetime.datetime(2022, 5, 27, 11, 29, 54, tzinfo=tzlocal())},\n                                              \'waiting\': None}}],\n            \'host_ip\': \'10.182.212.44\',\n            \'init_container_statuses\': None,\n            \'message\': None,\n            \'nominated_node_name\': None,\n            \'phase\': \'Failed\',\n            \'pod_ip\': \'172.24.35.219\',\n            \'qos_class\': \'Guaranteed\',\n            \'reason\': None,\n            \'start_time\': datetime.datetime(2022, 5, 27, 11, 29, 31, tzinfo=tzlocal())}}\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1165, in _run_raw_task\n    self._prepare_and_execute_task_with_callbacks(context, task)\n  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1283, in _prepare_and_execute_task_with_callbacks\n    result = self._execute_task(context, task_copy)\n  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1313, in _execute_task\n    result = task_copy.execute(context=context)\n  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py", line 374, in execute\n    raise AirflowException(f\'Pod Launching failed: {ex}\')\nairflow.exceptions.AirflowException: Pod Launching failed: Pod db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82 returned a failure: {\'api_version\': \'v1\',\n \'kind\': \'Pod\',\n \'metadata\': {\'annotations\': None,\n              \'cluster_name\': None,\n              \'creation_timestamp\': datetime.datetime(2022, 5, 27, 11, 29, 30, tzinfo=tzlocal()),\n              \'deletion_grace_period_seconds\': None,\n              \'deletion_timestamp\': None,\n              \'finalizers\': None,\n              \'generate_name\': None,\n              \'generation\': None,\n              \'initializers\': None,\n              \'labels\': {\'airflow_version\': \'2.1.4\',\n                         \'dag_id\': \'db-to-db-valdation-engine\',\n                         \'execution_date\': \'2022-05-27T112927.9797480000-ed9f0cbd9\',\n                         \'kubernetes_pod_operator\': \'True\',\n                         \'task_id\': \'db_to_db_validation_automation\',\n                         \'try_number\': \'1\'},\n              \'managed_fields\': [{\'api_version\': \'v1\',\n                                  \'fields\': None,\n                                  \'manager\': \'OpenAPI-Generator\',\n                                  \'operation\': \'Update\',\n                                  \'time\': datetime.datetime(2022, 5, 27, 11, 29, 30, tzinfo=tzlocal())},\n                                 {\'api_version\': \'v1\',\n                                  \'fields\': None,\n                                  \'manager\': \'kubelet\',\n                                  \'operation\': \'Update\',\n                                  \'time\': datetime.datetime(2022, 5, 27, 11, 29, 56, tzinfo=tzlocal())}],\n              \'name\': \'db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\',\n              \'namespace\': \'ara-adapt-iceberg\',\n              \'owner_references\': None,\n              \'resource_version\': \'1423729336\',\n              \'self_link\': \'/api/v1/namespaces/ara-adapt-iceberg/pods/db-to-db-validation-dag.2d0bdb45d94c4762a7b826ebf6c55d82\',\n              \'uid\': \'75ff3d87-f763-4e21-8d96-47a15d41fff6\'},\n \'spec\': {\'active_deadline_seconds\': None,\n          \'affinity\': {\'node_affinity\': None,\n                       \'pod_affinity\': None,\n                       \'pod_anti_affinity\': None},\n          \'automount_service_account_token\': None,\n          \'containers\': [{\'args\': None,\n                          \'command\': None,\n                          \'env\': [{\'name\': \'etl_name\',\n                                   \'value\': \'EBS_FND_FLEX_VALUE_RULES_VL\',\n                                   \'value_from\': None},\n                                  {\'name\': \'bucket_type\',\n                                   \'value\': \'minio\',\n                                   \'value_from\': None},\n                                  {\'name\': \'test_script_file_path\',\n                                   \'value\': \'Unit Testing/Marlin/Automated \'\n                                            \'Test \'\n                                            \'Script/FND_FLEX_VALUE_RULES_VL.csv\',\n                                   \'value_from\': None},\n                                  {\'name\': \'etl_run_id\',\n                                   \'value\': \'EBS_FND_FLEX_VALUE_RULES_VL \'\n                                            \'9853_no_run_2022-05-27 \'\n                                            \'16:59:24.161137\',\n                                   \'value_from\': None},\n                                  {\'name\': \'db_config_json\',\n                                   \'value\': "{\'source_details\': {\'platform\': "\n                                            "\'onpremises\', \'name\': \'oracle\', "\n                                            "\'connection_details\': "\n                                            "{\'password\': \'supportrdonly321\', "\n                                            "\'username\': \'SUPPORTRDONLY\', "\n                                            "\'port\': \'1521\', \'host\': "\n                                            "\'ora-stage-ebs-r1-vip.vmware.com\', "\n                                            "\'service_name\': \'VMWSTAGE\', "\n                                            "\'drivername\': "\n                                            "\'oracle.jdbc.driver.OracleDriver\', "\n                                            "\'database\': \'oracle\'}, "\n                                            "\'additional_details\': {\'table "\n                                            "name\': \'\'}}, \'target_details\': "\n                                            "{\'platform\': \'onpremises\', "\n                                            "\'name\': \'presto\', "\n                                            "\'connection_details\': "\n                                            "{\'password\': "\n                                            "\'W.8u8bRe^d^!u1Y3^VP\', "\n                                            "\'username\': \'svc.presto_ob_dev\', "\n                                            "\'port\': \'8443\', \'host\': "\n                                            "\'prestosql-stg.tkg.vmware.com\', "\n                                            "\'database\': \'edw_tesseract\'}, "\n                                            "\'additional_details\': "\n                                            "{\'tablename\': {}}}, "\n                                            "\'output_details\': {\'platform\': "\n                                            "\'onpremises\', \'name\': "\n                                            "\'postgresql\', "\n                                            "\'connection_details\': "\n                                            "{\'password\': \'mysecretpassword\', "\n                                            "\'username\': \'postgres\', \'port\': "\n                                            "\'5432\', \'host\': \'10.168.3.214\', "\n                                            "\'database\': \'output_dataset\'}, "\n                                            "\'additional_details\': {\'table "\n                                            "name\': "\n                                            "\'db_to_db_validation_output_final\'}}, "\n                                            "\'metrics_details\': {\'platform\': "\n                                            "\'onpremises\', \'name\': "\n                                            "\'postgresql\', "\n                                            "\'connection_details\': "\n                                            "{\'password\': \'mysecretpassword\', "\n                                            "\'username\': \'postgres\', \'port\': "\n                                            "\'5432\', \'host\': \'10.168.3.214\', "\n                                            "\'database\': \'output_dataset\'}, "\n                                            "\'additional_details\': {\'table "\n                                            "name\': "\n                                            "\'metrics_db_to_db_final\'}}}",\n                                   \'value_from\': None},\n                                  {\'name\': \'bucket_config_json\',\n                                   \'value\': "{\'data_needed\': {\'source and "\n                                            "target\': "\n                                            "[\'source_and_target_table_should_be_of_same_length_and_values\', "\n                                            "\'source_and_target_table_should_be_of_same_length_and_values\', "\n                                            "\'source_and_target_table_should_be_of_same_length_and_values\', "\n                                            "\'source_and_target_should_match\', "\n                                            "\'source_and_target_should_match\', "\n                                            "\'source_and_target_should_match\', "\n                                            "\'each_source_and_target_query_should_fetch_some_data\', "\n                                            "\'scd_columns_validation\', "\n                                            "\'scd_source_target_validation\'], "\n                                            "\'only target\': "\n                                            "[\'target_query_should_not_return_any_rows\', "\n                                            "\'target_query_should_not_return_any_rows\', "\n                                            "\'target_table_query_should_fetch_some_data\', "\n                                            "\'target_table_query_should_fetch_some_data\', "\n                                            "\'table_query_should_fetch_n_number_of_records\', "\n                                            "\'table_query_should_fetch_n_number_of_records\', "\n                                            "\'target_query_should_not_return_any_rows\', "\n                                            "\'target_query_should_not_return_any_rows\', "\n                                            "\'target_query_should_not_return_any_rows\', "\n                                            "\'scd_target_validation\', "\n                                            "\'scd_target_validation\', "\n                                            "\'target_query_should_not_return_any_rows\', "\n                                            "\'target_query_should_not_contain_any_duplicate_rows\']}, "\n                                            "\'validation_buckets\': "\n                                            "{\'source_and_target_table_should_be_of_same_length_and_values\': "\n                                            "[\'DataCompleteness_Validation\', "\n                                            "\'StraightThrough_Column_Validation\', "\n                                            "\'Data_Transformation\'], "\n                                            "\'source_and_target_should_match\': "\n                                            "[\'Structure_Validation\', "\n                                            "\'Count_Validation\', "\n                                            "\'Numeric_Column\'], "\n                                            "\'each_source_and_target_query_should_fetch_some_data\': "\n                                            "[\'Data_Availability_Validation\'], "\n                                            "\'scd_columns_validation\': "\n                                            "[\'SCD_Validation\'], "\n                                            "\'target_query_should_not_return_any_rows\': "\n                                            "[\'Duplicate_Value_Check\', "\n                                            "\'Default_Value_Validation\', "\n                                            "\'Empty_Check\', "\n                                            "\'Empty_Header_Check\', "\n                                            "\'Null_Check\', "\n                                            "\'Primary_Key_Column\'], "\n                                            "\'target_table_query_should_fetch_some_data\': "\n                                            "[\'Date_Column_Validation\', "\n                                            "\'Date_Column\'], "\n                                            "\'table_query_should_fetch_n_number_of_records\': "\n                                            "[\'Count_Value_Validation\', "\n                                            "\'Specific_Row_Count_Check\'], "\n                                            "\'scd_source_target_validation\': "\n                                            "[\'SCD_Type_1_count_check\'], "\n                                            "\'scd_target_validation\': "\n                                            "[\'SCD_Type_1_Duplicate_Check\', "\n                                            "\'SCD_Type_1_Null_Value_Check\'], "\n                                            "\'target_query_should_not_contain_any_duplicate_rows\': "\n                                            "[\'Duplicate_Check_F2F\']}}",\n                                   \'value_from\': None},\n                                  {\'name\': \'user\',\n                                   \'value\': \'priyanka\',\n                                   \'value_from\': None},\n                                  {\'name\': \'conn_id\',\n                                   \'value\': \'47\',\n                                   \'value_from\': None},\n                                  {\'name\': \'pipeline_id\',\n                                   \'value\': \'9853\',\n                                   \'value_from\': None}],\n                          \'env_from\': None,\n                          \'image\': \'harbor-repo.vmware.com/tesseract_iceberg/adapt:db2dblatest\',\n                          \'image_pull_policy\': \'Always\',\n                          \'lifecycle\': None,\n                          \'liveness_probe\': None,\n                          \'name\': \'base\',\n                          \'ports\': None,\n                          \'readiness_probe\': None,\n                          \'resources\': {\'limits\': {\'cpu\': \'2\', \'memory\': \'5Gi\'},\n                                        \'requests\': {\'cpu\': \'2\',\n                                                     \'memory\': \'5Gi\'}},\n                          \'security_context\': None,\n                          \'stdin\': None,\n                          \'stdin_once\': None,\n                          \'termination_message_path\': \'/dev/termination-log\',\n                          \'termination_message_policy\': \'File\',\n                          \'tty\': None,\n                          \'volume_devices\': None,\n                          \'volume_mounts\': [{\'mount_path\': \'/var/run/secrets/kubernetes.io/serviceaccount\',\n                                             \'mount_propagation\': None,\n                                             \'name\': \'default-token-76pt8\',\n                                             \'read_only\': True,\n                                             \'sub_path\': None,\n                                             \'sub_path_expr\': None}],\n                          \'working_dir\': None}],\n          \'dns_config\': None,\n          \'dns_policy\': \'ClusterFirst\',\n          \'enable_service_links\': True,\n          \'host_aliases\': None,\n          \'host_ipc\': None,\n          \'host_network\': None,\n          \'host_pid\': None,\n          \'hostname\': None,\n          \'image_pull_secrets\': [{\'name\': \'adapt-secret\'}],\n          \'init_containers\': None,\n          \'node_name\': \'decc-ara-minion033\',\n          \'node_selector\': None,\n          \'preemption_policy\': \'PreemptLowerPriority\',\n          \'priority\': 0,\n          \'priority_class_name\': None,\n          \'readiness_gates\': None,\n          \'restart_policy\': \'Never\',\n          \'runtime_class_name\': None,\n          \'scheduler_name\': \'default-scheduler\',\n          \'security_context\': {\'fs_group\': None,\n                               \'run_as_group\': None,\n                               \'run_as_non_root\': None,\n                               \'run_as_user\': None,\n                               \'se_linux_options\': None,\n                               \'supplemental_groups\': None,\n                               \'sysctls\': None,\n                               \'windows_options\': None},\n          \'service_account\': \'default\',\n          \'service_account_name\': \'default\',\n          \'share_process_namespace\': None,\n          \'subdomain\': None,\n          \'termination_grace_period_seconds\': 30,\n          \'tolerations\': [{\'effect\': \'NoExecute\',\n                           \'key\': \'node.kubernetes.io/not-ready\',\n                           \'operator\': \'Exists\',\n                           \'toleration_seconds\': 300,\n                           \'value\': None},\n                          {\'effect\': \'NoExecute\',\n                           \'key\': \'node.kubernetes.io/unreachable\',\n                           \'operator\': \'Exists\',\n                           \'toleration_seconds\': 300,\n                           \'value\': None}],\n          \'volumes\': [{\'aws_elastic_block_store\': None,\n                       \'azure_disk\': None,\n                       \'azure_file\': None,\n                       \'cephfs\': None,\n                       \'cinder\': None,\n                       \'config_map\': None,\n                       \'csi\': None,\n                       \'downward_api\': None,\n                       \'empty_dir\': None,\n                       \'fc\': None,\n                       \'flex_volume\': None,\n                       \'flocker\': None,\n                       \'gce_persistent_disk\': None,\n                       \'git_repo\': None,\n                       \'glusterfs\': None,\n                       \'host_path\': None,\n                       \'iscsi\': None,\n                       \'name\': \'default-token-76pt8\',\n                       \'nfs\': None,\n                       \'persistent_volume_claim\': None,\n                       \'photon_persistent_disk\': None,\n                       \'portworx_volume\': None,\n                       \'projected\': None,\n                       \'quobyte\': None,\n                       \'rbd\': None,\n                       \'scale_io\': None,\n                       \'secret\': {\'default_mode\': 420,\n                                  \'items\': None,\n                                  \'optional\': None,\n                                  \'secret_name\': \'default-token-76pt8\'},\n                       \'storageos\': None,\n                       \'vsphere_volume\': None}]},\n \'status\': {\'conditions\': [{\'last_probe_time\': None,\n                            \'last_transition_time\': datetime.datetime(2022, 5, 27, 11, 29, 31, tzinfo=tzlocal()),\n                            \'message\': None,\n                            \'reason\': None,\n                            \'status\': \'True\',\n                            \'type\': \'Initialized\'},\n                           {\'last_probe_time\': None,\n                            \'last_transition_time\': datetime.datetime(2022, 5, 27, 11, 29, 56, tzinfo=tzlocal()),\n                            \'message\': \'containers with unready status: [base]\',\n                            \'reason\': \'ContainersNotReady\',\n                            \'status\': \'False\',\n                            \'type\': \'Ready\'},\n                           {\'last_probe_time\': None,\n                            \'last_transition_time\': datetime.datetime(2022, 5, 27, 11, 29, 56, tzinfo=tzlocal()),\n                            \'message\': \'containers with unready status: [base]\',\n                            \'reason\': \'ContainersNotReady\',\n                            \'status\': \'False\',\n                            \'type\': \'ContainersReady\'},\n                           {\'last_probe_time\': None,\n                            \'last_transition_time\': datetime.datetime(2022, 5, 27, 11, 29, 31, tzinfo=tzlocal()),\n                            \'message\': None,\n                            \'reason\': None,\n                            \'status\': \'True\',\n                            \'type\': \'PodScheduled\'}],\n            \'container_statuses\': [{\'container_id\': \'docker://5be95bb4f48b21dac746f488d7ea4e9cd70991785481149fb6bf3563d6cff37c\',\n                                    \'image\': \'harbor-repo.vmware.com/tesseract_iceberg/adapt:db2dblatest\',\n                                    \'image_id\': \'docker-pullable://harbor-repo.vmware.com/tesseract_iceberg/adapt@sha256:d467a80a1942cb13b5e7efbddd4fc30408c439ca5e477ecb25381f4ede6c69d6\',\n                                    \'last_state\': {\'running\': None,\n                                                   \'terminated\': None,\n                                                   \'waiting\': None},\n                                    \'name\': \'base\',\n                                    \'ready\': False,\n                                    \'restart_count\': 0,\n                                    \'state\': {\'running\': None,\n                                              \'terminated\': {\'container_id\': \'docker://5be95bb4f48b21dac746f488d7ea4e9cd70991785481149fb6bf3563d6cff37c\',\n                                                             \'exit_code\': 1,\n                                                             \'finished_at\': datetime.datetime(2022, 5, 27, 11, 29, 56, tzinfo=tzlocal()),\n                                                             \'message\': None,\n                                                             \'reason\': \'Error\',\n                                                             \'signal\': None,\n                                                             \'started_at\': datetime.datetime(2022, 5, 27, 11, 29, 54, tzinfo=tzlocal())},\n                                              \'waiting\': None}}],\n            \'host_ip\': \'10.182.212.44\',\n            \'init_container_statuses\': None,\n            \'message\': None,\n            \'nominated_node_name\': None,\n            \'phase\': \'Failed\',\n            \'pod_ip\': \'172.24.35.219\',\n            \'qos_class\': \'Guaranteed\',\n            \'reason\': None,\n            \'start_time\': datetime.datetime(2022, 5, 27, 11, 29, 31, tzinfo=tzlocal())}}\n[2022-05-27 11:29:58,303] {taskinstance.py:1513} INFO - Marking task as FAILED. dag_id=db-to-db-valdation-engine, task_id=db_to_db_validation_automation, execution_date=20220527T112927, start_date=20220527T112930, end_date=20220527T112958\n[2022-05-27 11:29:58,332] {local_task_job.py:151} INFO - Task exited with return code 1\n[2022-05-27 11:29:58,344] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check\n\n'